frontend/src/components/TableHeaderColumn.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <th scope="col" :class="{'text-left': true, resizable}" :style="{'width': storedWidth}">
+    <div :style="{'display': 'flex', 'justify-content': 'space-between'}">
+        <slot>
+            <span @click="reorderBy(orderKey)" style="user-select: none">
+            <span>{{header}}</span><img :src="getOrderIcon(orderKey)" width="16" height="16">
+            </span>
+        </slot>
+        <div :class="{'resizer': true, resizing: isResizing}" scope="col" v-if="resizable" @mousedown="startResize">
+        </div>
+    </div>
+    </th>
+</template>
+
+<script>
+export default {
+    name: 'TableHeaderColumn',
+    data() {
+        return {
+            isResizing: false
+        }
+    },
+    computed: {
+        storedWidth(){
+            let columnWidth = this.$store.getters.getColumnWidth(this.orderKey);
+            if(columnWidth){
+                return columnWidth;
+            }
+            return null;
+        }
+    },
+    props: { 
+        'header': String ,
+        'orderKey': String,
+        'resizable': Boolean,
+    },
+    methods: {
+        resize(event){
+            if(!this.isResizing) return;
+            let boundingBox = this.resizeColumn.getBoundingClientRect();
+            let newWidth = event.clientX - boundingBox.left;
+            this.resizeColumn.style.width = `${newWidth}px`;
+        },
+        startResize(){
+            this.isResizing = true;
+            this.resizeColumn = this.$el;
+            document.addEventListener('mousemove', this.resize);
+            document.addEventListener('mouseup', this.stopResize);
+        },
+        stopResize(event){
```

Comment:
```suggestion
        stopResize(){
```

Can't compile because variable is not used, besides that everything else works smoothly and as expected